lib/server/services/log/LogService.js

```diff
@@ -131,17 +145,36 @@ class LogService {
      * @return {Promise<Log[]>} the resulting logs
      */
     async _getAllByIds(logIds) {
-        const logQueryBuilder = dataSource.createQueryBuilder()
-            .where('id').oneOf(...logIds)
-            .include({ association: 'runs', attributes: ['id', 'runNumber'] })
-            .include({ association: 'lhcFills', attributes: ['fillNumber'] })
-            .include({ association: 'environments', attributes: ['id'] });
+        const logQueryBuilder = this._createFindQueryBuilder();
 
+        logQueryBuilder.where('id').oneOf(...logIds);
```

Comment:
I forgot to mention, but here you can chain the where: 
```js
const logQueryBuilder = this._createFindQueryBuilder()
    .where('id').oneOf(...logIds);
```
If you think this way is less clear, keep the one you have.

Reply:
I think the chaining is clear, done.